src/init.c

```diff
@@ -289,9 +289,18 @@ void segv_handler(int sig, siginfo_t *info, void *context)
         sigprocmask(SIG_UNBLOCK, &sset, NULL);
         jl_throw(jl_memory_exception);
     }
+#ifdef SEGV_EXCEPTION
+    else {
```

Comment:
SIGILL, SIGTINFO, SIGUSR1, SIGKILL I believe all go through here also. The last three should not be thrown as Julia errors